run.sh

```diff
@@ -0,0 +1,25 @@
+#! /usr/bin/env bash
```

Comment:
Maybe we should add basic instructions in the `README.md` to help the reader to run this?

Reply:
Yup that was my intention.